package.json

```diff
@@ -41,7 +42,9 @@
     "proxyquire": "1.7.4"
   },
   "dependencies": {
-    "request-promise": "^2.0.1"
+    "request-promise": "^2.0.1",
+    "@types/node": "^8.0.26",
```

Comment:
Does this need to be in "dependencies" rather than "devDependencies"?

Reply:
@type/node is required for the definition of Buffer used in `ow.actions` (to build actions from zip files). I think it has to be a dependency, but it only contains type definitions.